properties/service_fabric_nuget.targets

```diff
@@ -23,6 +24,7 @@
     <Message Text="Copying NuGet packages to drop folder ..." Importance="High"/>
     <ItemGroup>
       <Nupkgs Include="nuprojs\Microsoft.ServiceFabric.Actors\$(OutputPath)\Microsoft.ServiceFabric.Actors.*.nupkg"/>
+	    <Nupkgs Include="nuprojs\Microsoft.ServiceFabric.StartupServicesUtility\$(OutputPath)\Microsoft.ServiceFabric.StartupServicesUtility.*.nupkg"/>
```

Comment:
are you creating a new nuget ? I thought the xsd/dll were to be packaged in the SF_Internal nuget.

Reply:
xsd and dll are packed in SF_Internal nuget. Along with this there is a separate nuget as well which can be used if someone needs only this nuget.

Reply:
Who would need this dll and xsd apart from the WF-SDK for VS  ? This XSD is only meant to be used with VS.

Reply:
Taking this out from nuget.

Reply:
Fixed